features/steps/go_steps.rb

```diff
@@ -1,122 +1,41 @@
-require 'net/http'
+require 'os'
 
-Given("I set environment variable {string} to the app directory") do |key|
-  step("I set environment variable \"#{key}\" to \"/app/src/test/\"")
+When('I run {string}') do |scenario_name|
+  execute_command 'run-scenario', scenario_name
 end
 
-Given("I set environment variable {string} to the notify endpoint") do |key|
-  step("I set environment variable \"#{key}\" to \"http://#{current_ip}:#{MOCK_API_PORT}\"")
+When('I configure the maze endpoint') do
```

Comment:
I'd change the wording slightly to be from the perspective of the notifier and its functionality, as opposed to the test tool we are using:
```suggestion
When('I configure the base endpoint') do
```
This could easily wait until a later stage though.

Reply:
will change here